app/components/file/browse/display/file-list-display-file-type-filter.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from "@angular/core";
```

Comment:
That's a good question. I didn't run into that issue. I liked this approach because swapping between the tabs is instant since we only download once. A service side filter could also work BUT we'd have a similar issue w/ the first batch of files not actually containing any files...